RockLib.UniversalMemberAccessor/UniversalMemberAccessor.cs

```diff
@@ -2234,19 +2253,23 @@ public override int GetHashCode()
                 unchecked
                 {
                     var hashCode = _type.GetHashCode();
+#if NET48
                     hashCode = (hashCode * 397) ^ _name.GetHashCode();
+#else
+                    hashCode = (hashCode * 397) ^ _name.GetHashCode(StringComparison.Ordinal);
```

Comment:
Woops just saw this.  Since StringComparison.Ordinal was used above without an #if can we remove this?

Reply:
No, `GetHashCode()` for NET48 doesn't support any overloading like the other frameworks do.